<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_parear">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Parear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sobe">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sobe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_desce">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Desce</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_esquerda">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Esquerda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_direita">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DIreita</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_horario">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Horario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_antihorario">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Antihorario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_frente">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Frente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tras">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tras</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_iniciar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_desligar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Desligar</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_passo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Passo</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_passo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_passoAdd">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_passoMinus">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CalibMaxSD">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> ↤</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CalibMinSD">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> ↦</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CalibMinED">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> ↦</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CalibMaxED">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> ↤</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CalibMinFT">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>  ↧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CalibMaxFT">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> ↥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCalibChao">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrar Chão</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_tunningFB">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="sliderPosition">
    <number>127</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_tunningLR">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="sliderPosition">
    <number>127</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tunnig Rest LR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tunnig Rest FB</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_tunningLR">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>127</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_tunningFB">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>220</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>127</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SetRest">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>270</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Rest</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
